### DISCUSSION TITLE
how to $set multiple value to multiple field

### DISCUSSION THREAD

**Discussion by christmex** [filamentphp/filament#13732] [Help]
### Package

Form builder

### Package Version

V3.2.92

### How can we help you?

is it possible to set multi-value to multi-field, here's the example I'm trying to achieve, set multi-field based on when other fields update, like I have select, if I select then it will update the value to multi-field, for now, it works but only for 1 field, how can I set multiple, like this approach

```php
->afterStateUpdated(function(Set $set, $state){
        $customer = self::$PO = PoHeader::with('customer')->find($state);
        $set('customer',$customer->customer->company_name);
        $set('brand',$customer->customer->brand);
    })
```

thank you.

**Comment by leandrocfe** [comment]
Could you explain why isn't it working?

**Reply by christmex** [reply]
It only update one field,not all

**Reply by leandrocfe** [reply]
Is it updating `customer`? What happens if you have only $set('brand',..?
